src/viewport.cpp

```diff
@@ -934,15 +945,9 @@ void SpriteCollector::CollectVoxel(const Voxel *voxel, const XYZPoint16 &voxel_p
 			FenceType fence_type = GetFenceType(fences, edge);
 			if (fence_type != FENCE_TYPE_INVALID) {
 				DrawData dd;
-				dd.level = slice;
-				dd.z_height = voxel_pos.z;
+				dd.Set(slice, voxel_pos.z, (edge + 4 * this->orient + 1) % 4 < EDGE_SW ? SO_FENCE_BACK : SO_FENCE_FRONT,
+						this->sprites->GetFenceSprite(fence_type, edge, gslope, this->orient), north_point);
```

Comment:
Looks like the correct style is 2 additional tabs according to the "Methods and function implementation" section in the [CodeStyle](https://github.com/FreeRCT/FreeRCT/blob/wiki/CodeStyle.md):

"Simple statements like assignment or function calls that are very long can be split over multiple lines. In that case, add 2 additional TABs before the second and further lines to show the line is a continuation."